packages/main/src/MessageStrip.ts

```diff
@@ -108,6 +104,17 @@ class MessageStrip extends UI5Element {
 	})
 	design!: `${MessageStripDesign}`;
 
+	/**
+	 * Defines the color scheme of the component.
+	 * There are 10 predefined schemes.
+	 * To use one you can set a number from `"1"` to `"10"`. The `colorScheme` `"1"` will be set by default.
+	 *
+	 * @default "1"
+	 * @public
+	 */
+	@property({ defaultValue: "1" })
+	colorScheme!: string;
+
 	/**
 	 * Defines whether the MessageStrip will show an icon in the beginning.
 	 * You can directly provide an icon with the `icon` slot. Otherwise, the default icon for the type will be used.
```

Comment:
I would add a note in the hideIcon property or/and in the icon slot docu stating that:

> Keep in mind that if a `ColorSet1` and `ColorSet2` design is used, not default icon is present. Applications are responsible to provide a slotted icon or to apply the hideIcon property.

Reply:
Done